root/programs/VB/Frameworks/Infrastructure/Business/Business/MyApsBaseLogic.vb

```diff
@@ -0,0 +1,584 @@
+'*  ----------  ----------------  -------------------------------------------------
+'*  20xx/xx/xx  ＸＸ ＸＸ         新規作成（テンプレート）
+'*  2010/03/30  西野 大介         ログ フォーマットにメソッド名を追加
+'*  2010/09/24  西野 大介         共通引数クラス内にユーザ情報を格納したので
+'*  2010/09/24  西野 大介         Damクラス内にユーザ情報を格納したので
+'*  2011/01/24  西野 大介         上記コードが通るカバレージを修正
+'*  2011/01/31  西野 大介         Damがnullの場合は処理しないように修正
+'*  2012/02/09  西野 大介         OLEDB、ODBCのデータプロバイダ対応
+'*  2012/04/05  西野 大介         \n → \r\n 化
+'*  2012/06/18  西野  大介        OriginalStackTrace（ログ出力）の品質向上
+'*  2015/08/28  Sandeep           Implemented code to create two dams
+'*                                One dam for ABT asynchronous business task 
+'*                                Other dam for AMT asynchronous management task 
+'**********************************************************************************
+
+' デバッグ用
+Imports System.Diagnostics
+
+' メソッドの属性を取得
+Imports System.Reflection
+
+' System
+Imports System
+Imports System.IO
+Imports System.Data
+Imports System.Text
+Imports System.Collections
+Imports System.Collections.Generic
+
+' 業務フレームワーク
+Imports Touryo.Infrastructure.Business.Business
+Imports Touryo.Infrastructure.Business.Common
+Imports Touryo.Infrastructure.Business.Dao
+Imports Touryo.Infrastructure.Business.Exceptions
+Imports Touryo.Infrastructure.Business.Presentation
+Imports Touryo.Infrastructure.Business.Util
+
+' フレームワーク
+Imports Touryo.Infrastructure.Framework.Business
+Imports Touryo.Infrastructure.Framework.Common
+Imports Touryo.Infrastructure.Framework.Dao
+Imports Touryo.Infrastructure.Framework.Exceptions
+Imports Touryo.Infrastructure.Framework.Presentation
+Imports Touryo.Infrastructure.Framework.Util
+Imports Touryo.Infrastructure.Framework.Transmission
+
+' 部品
+Imports Touryo.Infrastructure.Public.Db
+Imports Touryo.Infrastructure.Public.IO
+Imports Touryo.Infrastructure.Public.Log
+Imports Touryo.Infrastructure.Public.Str
+Imports Touryo.Infrastructure.Public.Util
+
+Namespace Touryo.Infrastructure.Business.Business
+    ''' <summary>自動振り分け機能付き業務コード親クラス２（サーバ用）（テンプレート）</summary>
+    ''' <remarks>（オーバーライドして）自由に利用できる。</remarks>
+    Public MustInherit Class MyApsBaseLogic
+        Inherits BaseLogic
+        ''' <summary>性能測定</summary>
+        Private perfRec As PerformanceRecorder
+
+        ''' <summary>Dam key value for asynchronous　Business task</summary>
+        Private _damKeyforABT As String = "ABT"
+
+        ''' <summary>Get dam key property to access asynchronous　Business task</summary>
+        Protected ReadOnly Property DamKeyforABT() As [String]
+            Get
+                Return Me._damKeyforABT
+            End Get
+        End Property
+
+        ''' <summary>Dam key value for asynchronous management task</summary>
+        Private _damKeyforAMT As String = "AMT"
+
+        ''' <summary>Get dam key property to access asynchronous management task</summary>
+        Protected ReadOnly Property DamKeyforAMT() As [String]
```

Comment:
This problem has occurred by the conversion to VB from C# using the SharpDeveloper.

Currently, the base class 2 and the sample program has been converted by the SharpDeveloper. If you want to modify this entire problem, since, it is necessary to modify the conversion by SharpDeveloper. However, I believe this operation is not realistic.